stable/pipelines/README.md

```diff
@@ -81,6 +81,17 @@ kubectl create ns pipelines
 helm upgrade --install pipelines --namespace pipelines center/jfrog/pipelines -f pipelines/values-ingress.yaml -f pipelines/values-ingress-passwords.yaml
 ```
 
+### Special Upgrade Notes
+
+While upgrading from Pipelines 1.4.x to 1.5.x and above charts due to breaking rabbitmq (when `rabbitmq.enabled=true`) subchart changes please run,
+
+```bash
+$ kubectl delete statefulsets <old_statefulset_pipelines_name>
```

Comment:
@chukka Please add instructions on how to get these old names. 
It would be great if you also add the names that are generated by the charts, i.e. `<release-name>-rabbitmq-ha-0`

Reply:
@rahulsadanandan please check